public/app/features/dashboard/displayProfiles/components/DisplayProfileSettings.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactElement, useCallback } from 'react';
+import { css } from '@emotion/css';
+import { Checkbox, CollapsableSection, Field, RadioButtonGroup, useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { DisplayProfileId, getDisplayProfile } from '../profiles';
+import { customProfile } from '../profiles/custom';
+import { get, set } from 'lodash';
+import { DisplayProfileMode } from '../types';
+import { getConfig } from 'app/core/config';
+import { useDisplayProfileId, useOnChangeDisplayProfile } from '../state/hooks';
+
+type DisplayProfileSettingsProps = {};
+
+export function DisplayProfileSettings(props: DisplayProfileSettingsProps): ReactElement | null {
+  const styles = useStyles2(getStyles);
+  const currentProfile = useDisplayProfileId();
+  const profile = getDisplayProfile(currentProfile);
+  const onChangeProfile = useOnChangeDisplayProfile();
+
+  console.log('currentProfile', currentProfile);
+
+  const onChange = useCallback(
+    (id: DisplayProfileId) => {
+      const profileConfig = getDisplayProfile(id);
+
+      if (!profileConfig) {
+        return;
+      }
+
+      onChangeProfile(id);
+    },
+    [onChangeProfile]
+  );
+
+  const onChangeOption = useCallback(
+    (path: string) => {
+      const mode = get(customProfile, path) as DisplayProfileMode;
+
+      if (mode === DisplayProfileMode.default) {
+        set(customProfile, path, DisplayProfileMode.hidden);
+        onChangeProfile(DisplayProfileId.custom);
+        return;
+      }
+
+      set(customProfile, path, DisplayProfileMode.default);
+      onChangeProfile(DisplayProfileId.custom);
+    },
+    [onChangeProfile]
+  );
+
+  if (!getConfig().featureToggles.customKiosk) {
+    return null;
+  }
+
+  const configurableOptions: Array<SelectableValue<string>> = [
+    { value: 'sideMenu', label: 'Side Menu' },
+    { value: 'dashNav.timePicker', label: 'Time picker' },
+    { value: 'dashNav.title', label: 'Title' },
+    { value: 'dashNav.tvToggle', label: 'Kiosk mode button' },
+    { value: 'dashNav.addPanelToggle', label: 'Add panel button' },
+    { value: 'dashNav.snapshotToggle', label: 'Snapshot Button' },
+    { value: 'dashNav.starToggle', label: 'Star button' },
+    { value: 'dashNav.sharePanelToggle', label: 'Share Dashboard or Panel Button' },
+    { value: 'dashNav.dashboardSettingsToggle', label: 'Settings button' },
+    { value: 'dashNav.saveDashboardToggle', label: 'Save button' },
+  ];
```

Comment:
other than timePicker and title I do not see a use case (demand) for any of these other toggles. 

The thing I have seen requested are
* Hide panel menu 
* Hide top toolbar but still show time picker (So DashNav with only time picker)
* Show variables (Submenu) 

But might have missed some other requests.